skrub/tests/test_fuzzy_join.py

```diff
@@ -353,7 +404,27 @@ def test_multiple_keys() -> None:
         right_on=["str_1", "str_2", "int2"],
         numerical_match="number",
     )
-    assert fj_mixed.shape == (3, 8)
+    assert fj_mixed.shape == (3, 10)
+
+    # On mixed time and string keys
+    fj_mixed2 = fuzzy_join(
```

Comment:
Same remarks here (maybe you can use Copilot or GPT to generate the tests for you?)

Reply:
ChatGPT says:

> Both Copilot and GPT can assist you in generating tests, but they have different capabilities and approaches.
> 
> Copilot is an AI-powered code completion tool that helps you write code more efficiently. It can generate code suggestions based on the context and patterns it has learned from large code repositories. Copilot is particularly useful for generating code snippets, completing function calls, or providing boilerplate code.
> 
> On the other hand, GPT, like ChatGPT, is a large language model that can understand and generate human-like text. It can assist with generating test cases, explaining code concepts, providing code samples, and answering questions about programming.

It's done.
:)